diff --git a/Library/OcCpuLib/OcCpuLib.c b/Library/OcCpuLib/OcCpuLib.c
index b564af2e..329a3407 100755
--- a/Library/OcCpuLib/OcCpuLib.c
+++ b/Library/OcCpuLib/OcCpuLib.c
@@ -745,7 +745,7 @@ OcCpuScanProcessor (
 
     Cpu->ExtFeatures = LShiftU64 (Cpu->CpuidExtSigEcx.Uint32, 32) | Cpu->CpuidExtSigEdx.Uint32;
   }
-
+#ifndef CLOVER_BUILD
   DEBUG ((DEBUG_INFO, "OCCPU: Found %a\n", Cpu->BrandString));
 
   DEBUG ((
@@ -760,7 +760,7 @@ OcCpuScanProcessor (
     Cpu->ExtFamily,
     Cpu->MicrocodeRevision
     ));
-
+#endif
   Cpu->CPUFrequencyFromVMT = InternalCalculateVMTFrequency (
     &Cpu->FSBFrequency,
     &Cpu->Hypervisor
@@ -821,7 +821,7 @@ OcCpuScanProcessor (
   } else if (Cpu->ExternalClock >= 265 && Cpu->ExternalClock <= 267) {
     Cpu->ExternalClock = 266;
   }
-
+#ifndef CLOVER_BUILD
   DEBUG ((
     DEBUG_INFO,
     "OCCPU: CPUFrequencyFromTSC %11LuHz %5LuMHz\n",
@@ -850,6 +850,7 @@ OcCpuScanProcessor (
     Cpu->CoreCount,
     Cpu->ThreadCount
     ));
+#endif
 }
 
 VOID
diff --git a/Platform/OpenCore/OpenCoreMisc.c b/Platform/OpenCore/OpenCoreMisc.c
index c893cb6c..108f1635 100644
--- a/Platform/OpenCore/OpenCoreMisc.c
+++ b/Platform/OpenCore/OpenCoreMisc.c
@@ -509,12 +509,13 @@ OcMiscEarlyInit (
   IN  OC_RSA_PUBLIC_KEY  *VaultKey  OPTIONAL
   )
 {
-  EFI_STATUS                Status;
-  EFI_TIME                  BootTime;
   CONST CHAR8               *AsciiVault;
   OCS_VAULT_MODE            Vault;
 
 #ifndef CLOVER_BUILD
+  EFI_STATUS                Status;
+  EFI_TIME                  BootTime;
+
   Status = ClOcReadConfigurationFile(Storage, OPEN_CORE_CONFIG_PATH, Config);
   if (EFI_ERROR (Status)) return EFI_UNSUPPORTED;
 #endif
@@ -546,9 +547,11 @@ OcMiscEarlyInit (
     CpuDeadLoop ();
     return EFI_SECURITY_VIOLATION; ///< Should be unreachable.
   }
-
+#ifndef CLOVER_BUILD
   DEBUG ((DEBUG_INFO, "OC: Watchdog status is %d\n", Config->Misc.Debug.DisableWatchDog == FALSE));
-
+#else
+  Config->Misc.Debug.DisableWatchDog = TRUE;
+#endif
   if (Config->Misc.Debug.DisableWatchDog) {
     //
     // boot.efi kills watchdog only in FV2 UI.
@@ -577,7 +580,7 @@ OcMiscEarlyInit (
     Storage->HasVault,
     VaultKey != NULL
     ));
-
+#ifndef CLOVER_BUILD
   Status = gRT->GetTime (&BootTime, NULL);
   if (!EFI_ERROR (Status)) {
     DEBUG ((
@@ -597,7 +600,7 @@ OcMiscEarlyInit (
       Status
       ));
   }
-
+#endif
   return EFI_SUCCESS;
 }
 
